src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -79,6 +79,7 @@
     <Compile Include="System\Text\Json\Serialization\Converters\JsonValueConverterUInt16.cs" />
     <Compile Include="System\Text\Json\Serialization\Converters\JsonValueConverterUInt32.cs" />
     <Compile Include="System\Text\Json\Serialization\Converters\JsonValueConverterUInt64.cs" />
+    <Compile Include="System\Text\Json\Serialization\Converters\JsonValueConverterUri.cs" />
```

Comment:
FYI we eventually need to rename all of the files from "JsonValueConverter*" to just "JsonConverter*" to match file:classname. I added that task to [issue ](https://github.com/dotnet/corefx/issues/38408).